src/polygons.jl

```diff
@@ -775,6 +775,76 @@ function polyfit(plist::Array{Point,1}, npoints = 30)
     return resultpoly
 end
 
+"""
+    bspline(controlPoints::Array{Point,1}, npoints; degree=3, clamped=true)
+
```

Comment:
If this creates a polygon, the name could start with `poly` to help people  find it: `polyspline`? `polybspline` and to be consistent with all the other ones. But `bspline` could be OK too...

`controlpoints` or `control_points` is more Julian than camel-ly`controllPoints` :)

Reply:
`polybspline` makes sense since we're discussing adding other spline types I guess. Also ugh I yeah you're right for some reason the Julia community hates capital letters lol